src/SphericalCollapse/spherical_collapse.cpp

```diff
@@ -114,8 +126,7 @@ template <> void RadhydroSimulation<CollapseProblem>::ComputeDerivedVar(int lev,
 		auto const &phi_data_ref = phi[lev].const_arrays();
 		auto output = mf.arrays();
 
-		amrex::ParallelFor(
-		    mf, [=] AMREX_GPU_DEVICE(int bx, int i, int j, int k) noexcept { output[bx](i, j, k, ncomp) = phi_data_ref[bx](i, j, k, ncomp); });
+		amrex::ParallelFor(mf, [=] AMREX_GPU_DEVICE(int bx, int i, int j, int k) noexcept { output[bx](i, j, k, ncomp) = phi_data_ref[bx](i, j, k); });
 	}
```

Comment:
what changed here? just some indentation?

Reply:
You referenced a non-existent component of `phi[lev]`. Since `phi[lev]` only has one component, you can omit the component index in the array and just give it the $(i,j,k)$ cell indices.